sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/SentimentSkill.java

```diff
@@ -24,6 +25,7 @@
 @JsonTypeName("#Microsoft.Skills.Text.SentimentSkill")
 @Fluent
 public final class SentimentSkill extends SearchIndexerSkill {
+    @JsonIgnore
```

Comment:
This is a good one! How did you find this? Should we add this in our code quality checks - not have ClientLogger w/o JsonIgnore in our models.

Reply:
Tests against a live Search service failed 😞